sympy/ntheory/ec.py

```diff
@@ -1,15 +1,16 @@
-from sympy.abc import x, y
+from sympy import symbols
 from sympy.core.compatibility import as_int, is_sequence
 from sympy.core.numbers import oo
 from sympy.core.relational import Eq
-from sympy.polys.domains import FiniteField, QQ, RationalField
+from sympy.polys.domains import FiniteField, QQ, RationalField, FF
 from sympy.solvers.solvers import solve
-
+from sympy import sympify
 from .factor_ import divisors
-from .residue_ntheory import sqrt_mod
+from .residue_ntheory import quadratic_congruence
 
+x, y = symbols('x y')
```

Comment:
These symbols should not be global. It they are needed in some class or function, they should be defined only there.